git-protocol/tests/fetch/mod.rs

```diff
@@ -175,9 +160,9 @@ mod blocking_io {
             mut input: impl io::BufRead,
             _progress: impl Progress,
             _refs: &[Ref],
-            previous: &Response,
+            response: &Response,
```

Comment:
Sometimes it's called `previous_result`, sometimes `previous`, I think I will call it `previous_response` everywhere 😁.